package.json

```
@@ -29,27 +29,27 @@
     "check-missing-license-data": "run-func build/yarn-scripts.ts checkMissingLicenseData",
     "check-license-compatibility": "run-func build/yarn-scripts.ts checkLicenseCompatibility",
     "clean": "yarn nodeclean && yarn scalaclean",
-    "precompile": "yarn nodeclean && yarn gen-version-ts && yarn webpack:svelte && yarn compile:tdmlEditorJS",
+    "precompile": "yarn nodeclean && yarn gen-version-ts && yarn svelte:build && yarn compile:tdmlEditorJS",
     "compile": "tsc -p ./ && yarn sbt",
     "lint": "yarn prettier src -c",
     "lint:fix": "yarn prettier src -w",
     "prewatch": "yarn gen-version-ts && yarn sbt",
     "watch": "run-func build/yarn-scripts.ts watch",
-    "watch:svelte": "yarn webpack --env development=true -w -c ./src/svelte/webpack.config.js",
+    "watch:svelte": "cd src/svelte && DEBUG_DATAEDITOR=on vite --config ./vite.config.mjs build --mode development --watch",
     "watch:svelte-tests": "mocha -r ts-node/register -w ./src/svelte/tests/**/*.test.ts",
     "watch:tdmlEditorJS": "esbuild src/tdmlEditor/webview/webview.js --outfile=dist/views/tdmlEditor/webview/webview.js --bundle --platform=node --format=cjs --watch",
-    "webpack": "webpack --mode production --config ./webpack/ext-dev.webpack.config.js",
-    "webpack:pkg": "webpack --mode production --config ./webpack/ext-package.webpack.config.js",
-    "webpack:svelte": "webpack -c ./src/svelte/webpack.config.js",
     "compile:tdmlEditorJS": "esbuild src/tdmlEditor/webview/webview.js --outfile=dist/views/tdmlEditor/webview/webview.js --bundle --platform=node --format=cjs",
-    "prepackage": "yarn install && yarn compile && yarn webpack:pkg",
+    "prepackage": "yarn install && yarn compile && yarn vite:pkg",
     "package": "run-func build/yarn-scripts.ts package && yarn --cwd dist/package install && yarn --cwd dist/package vsce package --out ../../",
-    "pretest": "yarn compile && yarn webpack",
+    "pretest": "yarn compile && yarn vite:pkg",
```

```suggestion
    "pretest": "yarn compile && yarn vite:dev",
```

Using `yarn vite:pkg` causes the test not to be able to run properly as it adds additional folder/file structure. When using `yarn vite:dev` everything should run fine.